### PR TITLE
feat: Add order custom forms in attendee details for text type

### PR DESCRIPTION
**Pull request by dreadpool2**
First steps #383 
videotogif_2018 08 03_22 19 40

### CODE REVIEW

**Comment by dreadpool2** [comment]
@srv-twry @iamareebjamal @simarsingh24 @nikit19 Please review

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt [hunk 1]
```diff
@@ -269,6 +271,22 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                 }))
     }
 
+    fun getCustomFormsForAttendees(eventId: Long) {
+        val filter = "[{\"name\":\"form\",\"op\":\"ilike\",\"val\":\"order\"}]"
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt]: why `ilike` ?

File: app/src/main/res/layout/fragment_attendee.xml [hunk 2]
```diff
@@ -396,6 +396,8 @@
                 </android.support.design.widget.TextInputLayout>
             </LinearLayout>
 
+            <include layout="@layout/content_attendee_information"/>
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_attendee.xml]: it is just a textview.  just add it here directly

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -78,6 +78,7 @@
 
     <!--ticket details-->
     <string name="your_info">Your Info</string>
+    <string name="more_attendee_info">More Attendee Information</string>
```
Comment (nikit19) [inline app/src/main/res/values/strings.xml]: use something more appropriate. see what Evenrbrite uses

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 4]
```diff
@@ -272,6 +276,19 @@ class AttendeeFragment : Fragment() {
                 redirectToLogin()
             }
 
+            //Custom forms fetch
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: either remove or add a meaningful comment. the method name is self explanatory IMO

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: Will remove it

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt (hunk 1 above)
Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt]: Changed to case sensitive

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (dreadpool2) [reply app/src/main/res/values/strings.xml]: Changed it to `Other Information`

**Comment by dreadpool2** [comment]
PR Updated

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt [hunk 5]
```diff
@@ -269,6 +271,22 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                 }))
     }
 
+    fun getCustomFormsForAttendees(eventId: Long) {
+        val filter = "[{\"name\":\"form\",\"op\":\"like\",\"val\":\"order\"}]"
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt]: why like? are you comparing values?

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt]: The filter requires a `op` value actually. So I thought of using `like`.

Comment (nikit19) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt]: That is why the equality operator should be used

File: app/src/main/res/layout/fragment_attendee.xml [hunk 6]
```diff
@@ -407,6 +425,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
+                android:enabled="false"
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_attendee.xml]: why?

Comment (dreadpool2) [reply app/src/main/res/layout/fragment_attendee.xml]: Will get enabled once loading of custom forms are done

**Pull request merged**